db/migrations/20231019082413_invitations.sql

```diff
@@ -0,0 +1,16 @@
+-- 2023-10-19 08:24:13 : invitations
+
+CREATE TABLE "Invitation"
+(
+    "Id"          UUID          NOT NULL,
+    "Invitee"     VARCHAR(32)   NOT NULL,
```

Comment:
Our UserIds are, unfortunately in my mind, the email of people.
I would have liked GUID, and started with that, but we cannot.

Reply:
UserIds are VARCHAR(255)
<img width="350" alt="image" src="https://github.com/dfds/selfservice-api/assets/5738476/f409ab5c-963d-49ef-8a83-dafbdefda6b0">

I suggest that we make a foreign key constraint to that ID with cascade delete like we do for membership members
<img width="982" alt="image" src="https://github.com/dfds/selfservice-api/assets/5738476/9f55bd4f-64c7-40d8-b515-81dc9faeb2b9">

If you don't want to do that, you have to add removal of invites in the membership cleaner

Reply:
The only reason I didn't do `VARCHAR(255)` is because <xuaz>@dfds.com will never be that long.
I'll change the varchar to 255 for consistency's sake and add the constraint, as that is a good idea for sure